docs-source/content/faq/coherence-requirements.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Coherence Requirements"
+date: 2019-08-12T12:41:38-04:00
+draft: false
+---
+
+If you are running Coherence on Kubernetes, either inside a WebLogic domain
+or standalone, then there are some additional requirements to make sure
+that Coherence can form clusters. 
+
+Note that some Fusion Middleware products, like SOA Suite, use Coherence
+and so these requirements apply to them.
+
+#### Unicast and Well Known Address 
+When the first Coherence process starts, it will form a cluster.  The next
+Coherence process to start (i.e. in a different pod) will use UDP to try
+to contact the master.  Most Kubernetes overlay network providers do not
+support multicast, so you should configure Coherence to use unicast and 
+provide a "well known address (WKA)" so that members can find the master. 
+This is typically done by using the cluster address, i.e. the address of
+the Kubernetes service that is pointing to all cluster members.
+
+{{% notice note %}}
+If you create your domain using the samples provided in this project, the
+cluster will be able to form; you do not need to manually specify WKA.
+However, the information is included below for completeness.
+{{% /notice %}}
+
+For example, suppose you have a domain called `domain1` and in that domain
+you have a cluster called `cluster1` and Coherence is running in that
+cluster.  You would therefore use the cluster service that is created by
+the operator as the well known address.  This service, in this example, 
+would be called `domain1-cluster-cluster1` and follows the pattern:
+
+```
+domainUID  "-cluster-"  cluster name
+```
+
+Where `-cluster-` is a string literal. Note also that the operator
+normalizes the domainUID and the cluster name if necessary so that they
+are DNS compatible, for example all lower case, underscores converted 
+to hyphens.
+
+The Coherence well known addresses are provided in a variable named
+`coherence.wka` as shown in the example below:
+
+```
+-Dcoherence.wka=domain1-cluster-cluster1
```

Comment:
There is no need to specify coherence.wka since the WKA list is automatically generated by the managed services that comprise the Coherence cluster.

Reply:
yeah but if you are running it outside a domain you do need it... so i kinda wanted to leave it in there just in case...

Reply:
Wouldn't we just point them to Coherence Operator doc  if they are are not running in the context of a domain?

Reply:
Also, if you leave it, then please say it is not needed for WebLogic operator based domains.

Reply:
please check new commit and see if that works better

Reply:
That looks better (see my slack about service annotation)